ui/modal/password_modal.go

```diff
@@ -118,9 +122,22 @@ func (pm *PasswordModal) SetError(err string) {
 	}
 }
 
+func (pm *PasswordModal) handleEnterKeypress() {
+	select {
+	case event := <-pm.keyEvent:
+		if (event.Name == key.NameReturn || event.Name == key.NameEnter) && event.State == key.Press {
+			pm.enterKeyPressed = true
+			return
+		}
+	default:
+	}
+}
+
```

Comment:
Not needed as we have `handleSubmitEvent` which handles submit events for modals with editors.